src/System.Runtime.Extensions/tests/System/Math.cs

```diff
@@ -751,6 +745,19 @@ public static void Ceiling_Double(double value, double expectedResult, double al
             AssertEqual(expectedResult, Math.Ceiling(value), allowedVariance);
         }
 
+        [Theory]
+        [InlineData(-0.78539816339744831,    -0.0,                     0.0)]    // value: -(pi / 4)
+        [InlineData(-0.70710678118654752,    -0.0,                     0.0)]    // value: -(1 / sqrt(2))
+        [InlineData(-0.69314718055994531,    -0.0,                     0.0)]    // value: -(ln(2))
+        [InlineData(-0.63661977236758134,    -0.0,                     0.0)]    // value: -(2 / pi)
+        [InlineData(-0.43429448190325183,    -0.0,                     0.0)]    // value: -(log10(e))
+        [InlineData(-0.31830988618379067,    -0.0,                     0.0)]    // value: -(1 / pi)
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
(minor) would a comment here help a reader understand why these tests are skipped on full framework?

Reply:
Possibly, but we I don't see comments in many other places clarifying the netfx vs netcore test differences.
If we feel that it would be more generally useful, I'll update in a follow up PR.